packages/ndla-ui/src/LinkBlock/LinkBlock.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import styled from '@emotion/styled';
+import SafeLink from '@ndla/safelink';
+import { Forward, CalendarEd } from '@ndla/icons/common';
+import { breakpoints, colors, spacing, mq } from '@ndla/core';
+import { Heading } from '..';
```

Comment:
Liker bedre dette formatet.
```suggestion
import Heading from '../Heading';
```